### PR TITLE
updated changelog for set_value() third argument.

### PR DESCRIPTION
**Pull request by avenirer**
according to the pull request #3562

### CODE REVIEW

File: user_guide_src/source/changelog.rst [hunk 1]
```diff
@@ -105,6 +105,7 @@ Release Date: Not Released
 
       - :php:func:`form_dropdown()` will now also take an array for unity with other form helpers.
       - :php:func:`form_prep()` is now DEPRECATED and only acts as an alias for :doc:`common function <general/common_functions>` :php:func:`html_escape()`.
+      - :php:func:`set_value()` will also receive a boolean third argument for outputing the html escaped value (TRUE) or not (FALSE). Default set to FALSE.
```
Comment (narfbg) [inline user_guide_src/source/changelog.rst]: The default is TRUE, not FALSE.

And you might as well make it a bit shorter ...

    - :php:func:`set_value()` will now also accept a third argument, allowing to turn off HTML escaping of the value.

**Comment by narfbg** [comment]
Also, please just push that to #3562 ... it shouldn't be a separate pull request.

**Comment by avenirer** [comment]
I am really sorry, but I don't know how to push a pull request to another pull request :(. If it's ok with you I can create a new pull request for the documentation and changelog.

**Comment by narfbg** [comment]
You don't push a pull request to another pull request ... you push a commit, just like you've made the requested changes to system/helpers/form_helper.php, nothing is stopping you from adding changes to other files.

**Comment by avenirer** [comment]
Yeah... still don't have a clue on how to do it.

**Pull request closed without merging**